package.json

```diff
@@ -7,6 +7,7 @@
   "private": true,
   "//": "Use gulp for dev. It's cross platform.",
   "scripts": {
+    "build": "gulp build",
```

Comment:
I would like to avoid package.json scripts "pollution" because everything goes through gulpfile. Until we remove gulp (I still believe gulp is better then plain npm).